jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -147,6 +148,22 @@ public ResourceCollection(String csvResources)
         return _resources;
     }
 
+    /**
+     * Sets the resource collection's resources.
+     *
+     * @param res the resources to set
+     */
+    public void setResources(List<Resource> res)
```

Comment:
Maybe it would be better to change _resources from an array to a List<Resource>?

Reply:
I've no probs with that, but it has to be pass by value semantics and not pass by reference, so the list will be copied no matter what.

Reply:
Done.